Makefile

```diff
@@ -89,6 +89,11 @@ cri-containerd:
 cc-containerd:
 	bash integration/containerd/confidential/run_tests.sh
 
+# Run the Confidential Containers tests for kubernetes.
+cc-kubernetes:
+	export K8S_TEST_UNION="integration/kubernetes/confidential/agent_image.bats"
+	bash integration/kubernetes/confidential/run_kubernetes_tests.sh
```

Comment:
Hi @GeorginaKin , I've manually created a job on Jenkins so we can test this PR in the CI environment: http://jenkins.katacontainers.io/job/wainer-tests-CCv0-ubuntu-20.04-x86_64-CC_CRI_CONTAINERD_K8S

And, as you can see in http://jenkins.katacontainers.io/job/wainer-tests-CCv0-ubuntu-20.04-x86_64-CC_CRI_CONTAINERD_K8S/2/console it failed. The path `integration/kubernetes/confidential/run_kubernetes_tests.sh` doesn't exist, you will need to fix that. Also the K8S_TEST_UNION should be relative (I guess) to the `integration/kubernetes` directory, so it should be `confidential/run_kubernetes_tests.sh`

Reply:
Hey @wainersm thank you for the review again! We've fixed the path issue of `run_kubernetes_tests.sh` not being found and also made the path relative for `K8S_TEST_UNION` too 🙂